### PR TITLE
Added support for setting customized headers

### PR DESCRIPTION
**Pull request by shuaichang**
As title suggested.

# Testing Done
Added unit tests by setting invocation type header through request.setHeader

### CODE REVIEW

File: src/main/java/com/aliyuncs/fc/http/HttpRequest.java [hunk 1]
```diff
@@ -71,4 +73,14 @@ public HttpURLConnection getHttpConnection(String urls, byte[] content,
         }
         return httpConn;
     }
+
+    public void setHeader(String key, String value) {
+        Preconditions.checkArgument(!Strings.isNullOrEmpty(key), "Header key cannot be blank");
+        Preconditions.checkArgument(!Strings.isNullOrEmpty(key), "Header value cannot be blank");
```
Comment (rockuw) [inline src/main/java/com/aliyuncs/fc/http/HttpRequest.java]: key => value

Comment (rockuw) [inline src/main/java/com/aliyuncs/fc/http/HttpRequest.java]: 确认一下：如果这里set了一些类似于`Date`, `Authorization`这样的header，会被我们覆盖吗？

Comment (shuaichang) [inline src/main/java/com/aliyuncs/fc/http/HttpRequest.java]: key => value: fixed

> 确认一下：如果这里set了一些类似于Date, Authorization这样的header，会被我们覆盖吗？
是的，这些的header如果被set了会被override
https://github.com/aliyun/fc-java-sdk/blob/9dc3aca581f3e28f8c9ee486aff4fefa517410e0/src/main/java/com/aliyuncs/fc/client/DefaultFcClient.java

**Comment by rockuw** [comment]
+1

**Pull request merged**